packages/slice-machine/server/src/api/index.ts

```diff
@@ -107,12 +91,10 @@ router.post(
   "/custom-screenshot",
   // eslint-disable-next-line @typescript-eslint/no-misused-promises, @typescript-eslint/no-misused-promises, @typescript-eslint/no-explicit-any
   async function (req: any, res: express.Response): Promise<Express.Response> {
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-member-access
+    /* eslint-disable */
     const payload = await customScreenshot(req.files.file, req.body);
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-    if (payload.err) {
```

Comment:
can this err?

Reply:
nop the payload cannot contain the key err